CMakeLists.txt

```diff
@@ -17,23 +17,20 @@ ELSE()
 ENDIF()
 
 # The source of the library we want to build
-add_library(${PROJECT_NAME} src/generated/64/plugins/src/StarterPlugin/StarterPlugin.c)
+add_library(${PROJECT_NAME} SHARED src/generated/64/plugins/src/StarterPlugin/StarterPlugin.c)
 
+#Add the VM dependency
 add_subdirectory(dependencies/pharo-vm)
+target_link_libraries(${PROJECT_NAME} PharoVMCore)
+target_include_directories(${PROJECT_NAME}
+    PRIVATE
+    ${PharoVM_BINARY_DIR}/build/include/pharovm/
+    ${PharoVM_BINARY_DIR}/build/include/pharovm/
+    ${PharoVM_SOURCE_DIR}/include/pharovm/
+    ${PharoVM_SOURCE_DIR}/extracted/vm/include/common
+    ${PharoVM_SOURCE_DIR}/extracted/vm/include/osx)
```

Comment:
I cleaned this a bit, but it's not right.
The Pharo VM is not exporting its header files (as PUBLIC).
If the VM exported its header files as public then all these lines should not be required !!

Reply:
In fact, I'm checking and the VM exports this correctly, so this should not be needed. I'll update the PR